core/domain/collection_services.py

```diff
@@ -42,6 +42,7 @@
     models.NAMES.collection, models.NAMES.user])
 memcache_services = models.Registry.import_memcache_services()
 search_services = models.Registry.import_search_services()
+datastore_services = models.Registry.import_datastore_services()
```

Comment:
Alphabetize (put this above memcache_services).

Reply:
ok.